.github/workflows/twin-validator-test.yaml

```diff
@@ -0,0 +1,20 @@
+name: Twin Validator Test
+
+permissions:
+  issues: write
+  pull-requests: write
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 */3 * * *"
+
+jobs:
+  run-twin-validator-test:
+    uses: ./.github/workflows/run-forge.yaml
+    secrets: inherit
+    with:
+      FORGE_NAMESPACE: forge-twin-validator
+      FORGE_CLUSTER_NAME: aptos-forge-big-1
```

Comment:
Can you add `FORGE_RUNNER_DURATION_SECS` here? May be just run this test for 15 mins.